### ISSUE TITLE
Sparse checkout doesn't work

### ISSUE THREAD

**Issue by ycollet** [git-lfs/git-lfs#2195]
Hello,

I installed lfs-test-server on a server. It works fine using regular lfs clone and checkout.
I use git-lfs 2.0.1 on windows and Linux.
On git bash, I try to do a sparse checkout
```
$ git clone --no-checkout git@myserver.com:MyRepo.git MyRepo2
$ git config --local core.sparsecheckout true
$ echo 'MyDirToImport/*' > .git/info/sparse-checkout
$ git read-tree -mu HEAD

batch request: FATAL: unknown git/gitolite command: 'git-lfs-authenticate MyRepo download': exit status 25
```

**Comment by technoweenie** [comment]
Git LFS only works with an HTTPS server. It uses a [custom `git-lfs-authenticate` command](https://github.com/git-lfs/git-lfs/blob/master/docs/api/server-discovery.md#ssh) for SSH support. I don't know of any `git-lfs-authenticate` support in gitolite though.

**Comment by ycollet** [comment]
OK, thanks for this information

**Comment by ycollet** [comment]
I found this:

https://github.com/HimaJyun/gitolite-lfs

----- Mail original -----
De: "risk danger olson" <notifications@github.com>
À: "git-lfs/git-lfs" <git-lfs@noreply.github.com>
Cc: "ycollet" <ycollette.nospam@free.fr>, "Author" <author@noreply.github.com>
Envoyé: Mardi 2 Mai 2017 15:11:28
Objet: Re: [git-lfs/git-lfs] Sparse checkout doesn't work (#2195)



Closed #2195 .
